package-overrides/npm/semantic-ui@1.11.6.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "wrench"            : "git://github.com/derekslife/wrench-js.git"
```

Comment:
Hi, 
it seems that the official npm for wrench js is based on github.com/ryanmcgrath/wrench-js
while the official Semantic UI package (https://github.com/Semantic-Org/Semantic-UI/blob/master/package.json) Is based on a fork (derekslife/wrench-js), which has no npm.